### ISSUE TITLE
Borrow missing from the tools directory

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6917]
### Discussed in https://github.com/ArctosDB/arctos/discussions/6916

<div type='discussions-op-text'>

<sup>Originally posted by **StefanieBond** November  2, 2023</sup>
For documenting an incoming loan, under transactions there was a search 'borrow' and create 'borrow'.  Now it is all folded under 'Search loans', and you can select type of loan 'borrow' and then you can 'create borrow'.  This works, but I wonder why the 'borrow button' up front disappeared.</div>

**Comment by mkoo** [comment]
Are you saying add Borrow: Create as a link to the Accessions, Loans, Permits  tile? (we can add Borrow to that too)?

But I thought "/borrow.cfm" was a subform?

what am I missing something? (or just call and explain to my tired brain....)

**Comment by dustymc** [comment]
Maybe two things going on here.

First, https://arctos.database.museum/directory.cfm

splat-F "borrow" --> nada
splat-F "transaction" --> nada

We list only some types of transactions - and that's on the heels of the whole 'don't call some part-based transactions "loans"' thing! It was 'transactions' for years, I don't think it's possible to list all the present and future types, listing 'nodes' conflicts with the aforementioned (but maybe that's dead now??), people keep getting lost in there, (https://github.com/ArctosDB/arctos/issues/5742 is another).

Second, there's no direct link to https://arctos.database.museum/borrow.cfm?action=new but there is to eg https://arctos.database.museum/accn.cfm?action=createForm.

**Comment by mkoo** [comment]
OK here's my proposed changes for the Accessions, Loans, Permits tile (which I can do). I will also update the info blurbs too -- am I missing anything?

| current | proposed |
| ---------| ------------ |
| Accession & Loan: Search | Any Transaction: Search (and update the info button)| 
| Accession: Bulkload Metadata | no change |
| Accession: Create | no change |
| Borrow: Create | add new |
| Data Loan: Bulk Unload Records | no change
| Data Loan: Bulkload Records | no change |
| Loan: Bulkload Metadata | no change |
| Loan: Create | no change |
| Loan: Item Bulkload | no change |
| Loan: Search | no change |
| Permit: Search and Create | no change |

**Comment by mkoo** [comment]
well there's some new ordering due to alphabet! But Borrow: Create is linked now
So splat+F away

anything else missing in this tile?

**Comment by dustymc** [comment]
Looks good to me, closing, @StefanieBond please feel free to re-open if we've misunderstood something.

**Comment by StefanieBond** [comment]
Looks like the <create borrow> goes to a blank page.

**Comment by dustymc** [comment]
Bah thanks - should be fixed but it will be caching for about an hour.

**Comment by StefanieBond** [comment]
It is working now! Thanks!
